neutron/assetlist.json

```diff
@@ -313,6 +313,55 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dydx/images/dydx.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dydx/images/dydx.svg"
       }
+    },
+    {
+      "description": "ASTRO on Neutron",
+      "denom_units": [
+        {
+          "denom": "ibc/5751B8BCDA688FD0A8EC0B292EEF1CDEAB4B766B63EC632778B196D317C40C3A",
+          "exponent": 0,
+          "aliases": [
+            "uastro"
+          ]
+        },
+        {
+          "denom": "astro",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/5751B8BCDA688FD0A8EC0B292EEF1CDEAB4B766B63EC632778B196D317C40C3A",
+      "name": "Astroport token",
+      "display": "astro",
+      "symbol": "ASTRO",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "terra2",
+            "base_denom": "cw20:terra1nsuqsk6kh58ulczatwev87ttq2z6r3pusulg9r24mfj2fvtzd4uq3exn26",
+            "channel_id": "channel-167"
+          },
+          "chain": {
+            "channel_id": "channel-5",
+            "path": "transfer/channel-5/cw20:terra1nsuqsk6kh58ulczatwev87ttq2z6r3pusulg9r24mfj2fvtzd4uq3exn26"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "terra2",
+            "base_denom": "cw20:terra1nsuqsk6kh58ulczatwev87ttq2z6r3pusulg9r24mfj2fvtzd4uq3exn26"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/astro.png",
```

Comment:
I don't know why the images were missing, so thanks for adding them. But can you please add versions that don't have padding so they don't look shrunk on various UIs?